packages/schematics/migrations/update-6-1-1/update-6-1-1.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  Rule,
+  Tree,
+  SchematicContext,
+  chain,
+  template,
+  apply,
+  mergeWith,
+  url
+} from '@angular-devkit/schematics';
+import { updateJsonInTree, readJsonInTree } from '../../src/utils/ast-utils';
+import { stripIndents } from '@angular-devkit/core/src/utils/literals';
+
+function displayInformation(_, context: SchematicContext) {
+  context.logger.info(stripIndents`
+    A global base karma config has been added at karma.conf.ts
+    
+    This file exports a karma config to be extend in each project
+  `);
+
+  context.logger.warn(stripIndents`
```

Comment:
change it to "info". It's not really a warning--it's just a piece of information.

A warning is something folks should act upon. They don't have to do anything in this case.